packages/runtime/container-runtime/src/summary/summaryGenerator.ts

```diff
@@ -184,8 +185,7 @@ export class SummarizeResultBuilder {
 /**
  * Errors type for errors hit during summary that may be retriable.
  */
-export class RetriableSummaryError extends LoggingError {
-	public readonly canRetry = this.retryAfterSeconds !== undefined;
+export class RetriableSummaryError extends LoggingError implements IRetriableSummaryError {
```

Comment:
There is nothing summary related in this error. We should either have ay summary related information in this error or should rename the class.